### DISCUSSION TITLE
Print a non-dynamic element higher than the terminal

### DISCUSSION THREAD

**Discussion by LordWhiro** [ArthurSonzogni/FTXUI#949] [Q&A] [answered]
Hello,
I'm willing to use FTXUI for the tables only. Non-dynamic (no loop, no renderer, no component).
But the "screen system" prevents me to display something higher than the terminal, to create a screen higher than the terminal, so that I can just scroll the terminal and see my whole table.

Example:
```cpp
#include <ftxui/dom/table.hpp>

int main()
{
    using namespace std;
    using namespace ftxui;
    Table table
    {
        [] ()
        {
            vector<vector<Element>> cells;
            vector<Element> row {text("POSITIVE INTEGERS") | hcenter};
            cells.push_back(row);

            for (int i = 1; i <= 100; ++i)
            {
                vector<Element> row {text(to_string(i))};
                cells.push_back(row);
            }

            return cells;
        }()
    };
    table.SelectRow(0).BorderBottom();
    table.SelectAll().Border();
    table.SelectAll().SeparatorVertical();

    // Simple display
    ftxui::Element element = table.Render();
    Screen screen = Screen::Create(Dimension::Fit(element));
    element->Render(screen);
    screen.Print();
    std::cout << std::endl;

    return EXIT_SUCCESS;
}
```
This only prints the 45 first rows. Can't I print the full table? With the 100 rows?
Thanks in advance for the answer.

**Comment by ArthurSonzogni** [comment, marked as answer]
Thanks!

This is related to issue:
https://github.com/ArthurSonzogni/FTXUI/issues/572

Would someone be interested to submit a pull request?
I think the API could be:
```cpp
Dimension::Fit(element, bool extend_beyond_screen = false)
```

**Reply by LordWhiro** [reply]
Thanks for the answer. I'm going to try to submit a PR. (my first one ever, on a free software)
Let's continue the talk in issue #572 :slightly_smiling_face:
